src/api/v1alpha1/workload.go

```diff
@@ -39,6 +40,8 @@ var AllWorkloads = []string{
 type Workload struct {
 	// For pure Pod, no object reference.
 	corev1.ObjectReference `json:"metadata"`
+	// Replicas of this workload.
+	Replicas appsv1.ReplicaSet `json:"replicas"`
```

Comment:
We don't need to capture ReplicaSet, we just need to capture Replica's count.

Reply:
Intention was to capture whole info regarding replicaset. So, anyone could use in future if required. But if we required only count then I can change it.

Reply:
During our debug session, we have seen that generated(KubeBuilder) `goharbor.io_assessmentreports.yaml` has increased by 2000 lines just by ReplicatSet spec.

Also in practical scenarios, I don't see any added value in Capturing a complete ReplicaSet.  Atleast I don't see value w.r.t Governor.

Also please update the  goharbor.io_assessmentreports.yaml with generated values using KubeBuilder.